include/tvm/topi/transform.h

```diff
@@ -37,6 +40,7 @@
 #include <string>
 #include <unordered_set>
 #include <vector>
+#include <bitset>
```

Comment:
Need to have a reason about bitset

Reply:
Bitset saves space and is an ideal structure for saving the alphabet in the input of einsum "subscript". When implementing the ParseEinsumInput, each alphabet is mapped to bitset and could be processed easily in next stage. Replacing bitset with int[] or vector seems a little redundant.